### PR TITLE
[docs] Add data-source section

### PR DESCRIPTION
**Pull request by VSevagen**
This commit adds the section about the supported
data sources in GrimoireLab, as well as how to
add them to GrimoireLab

Signed-off-by: sevagenv <sevagenv@gmail.com>

### CODE REVIEW

File: docs/data-sources/supported-sources.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
```
Comment (vchrombie) [inline docs/data-sources/supported-sources.md:1]: Can you rename the file as `supported.md`?

File: docs/data-sources/add-sources.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+$ docker-compose up -d
+```
+
+Give it some time to gather the data and after a while your dashboard and data
+should be ready at `http://localhost:5601`.
+
+![dashboard](./assets/dashboard.png)
+
+In the case you need to add another data source, refer to the
+[configuration]()
```
Comment (vchrombie) [inline docs/data-sources/add-sources.md:81]: `to the [configuration](docs/getting-started/configure/) section`

File: docs/data-sources/supported-sources.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  [remo](https://github.com/chaoss/grimoirelab-sirmordred#remo-)
+- **Social Networks**:
+  [Twitter](https://github.com/chaoss/grimoirelab-sirmordred#twitter-)
+- **Others**: [RSS](https://github.com/chaoss/grimoirelab-sirmordred#rss-),
+  [Cocom](https://github.com/chaoss/grimoirelab-sirmordred#cocom-),
+  [Colic](https://github.com/chaoss/grimoirelab-sirmordred#colic-),
+  [Weblate](https://github.com/chaoss/grimoirelab-sirmordred#weblate-),
+  [google_hits](https://github.com/chaoss/grimoirelab-sirmordred#google_hits-),
+  [Dockerdeps](https://github.com/chaoss/grimoirelab-sirmordred#dockerdeps-),
+  [dockersmells](https://github.com/chaoss/grimoirelab-sirmordred#dockersmells-)
```
Comment (vchrombie) [inline docs/data-sources/supported-sources.md:65]: Can you add one more para, after this, adding a reference to https://github.com/chaoss/grimoirelab/blob/master/community_components.md#other-data-sources?

**Review by vchrombie — CHANGES_REQUESTED** [review]
Most of the PR looks good, I have very few comments. Please check them.

**Comment by VSevagen** [comment]
@vchrombie I updated the PR. Could you check it out ?

**Review by vchrombie — APPROVED** [review]
LGTM

**Pull request merged**